content_schemas/formats/world_index.jsonnet

```diff
@@ -0,0 +1,73 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: [
+    "world_index"
```

Comment:
I like this, I guess if we used this consistently in other schemas we wouldn't have any new document type modifying almost every schema. Maybe a future improvement for the backburner somewhere.